### ISSUE TITLE
Uncaught Exception running the installer

### ISSUE THREAD

**Issue by MauJFernandezUVR** [anchorcms/anchor-cms#920]
I just cloned the repo and when running the installer, I get this error:

```
Uncaught Exception

syntax error, unexpected ')'

Origin

install/run.php on line 48

Trace

#0 [internal function]: System\Error::shutdown()
#1 {main}
```

**Comment by daviddarnes** [comment]
Which branch did you use? If you want to use anchor we recommend downloading it from the official website http://anchorcms.com

**Comment by fbrinker** [comment]
yeah there is a bug in the run.php - the PR https://github.com/anchorcms/anchor-cms/pull/916 has to be merged to fix this

**Comment by MauJFernandezUVR** [comment]
Sorry for the delay.
@daviddarnes Yeah, after that I downloaded it from the link and not cloning it and it's working, thanks.
@fbrinker Awesome, thanks.

You can close it if you want.

**Comment by daviddarnes** [comment]
@MauJFernandezUVR good to hear!
